cupyx/scipy/sparse/compressed.py

```diff
@@ -357,6 +358,16 @@ def __init__(self, arg1, shape=None, dtype=None, copy=False):
             if shape is None:
                 shape = arg1.shape
 
+        elif isinstance(arg1, tuple) and len(arg1) == 2:
+            # Note: This implementation is not efficeint, as it first
+            # constructs a sparse matrix with coo format, then converts it to
+            # compressed format.
+            sp_coo = coo.coo_matrix(arg1, shape=shape, dtype=dtype, copy=copy)
+            sp_compressed = getattr(sp_coo, 'to' + self.format)()
```

Comment:
```suggestion
            sp_compressed = sp_coo.asformat(self.format)
```
like the sparse case (`if base.issparse(arg1): x = arg1.asformat(self.format)`).